### PR TITLE
Add Permissions to PropertyViewSet

### PR DESCRIPTION
**Pull request by haneslinger**
@axelstudios @kflemin the create test just won't work. I don't think it's reading the args
```
django.db.utils.IntegrityError: null value in column "cycle_id" violates not-null constraint
DETAIL:  Failing row contains (171, null, null, null).
```
why?

### CODE REVIEW

**Comment by haneslinger** [comment]
@kflemin re-pinging you on these failing tests

File: seed/tests/test_property_views_viewset.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        self.login_as_child_member()
+        resp = self.client.patch(url, params, content_type='application/json')
+        assert resp.status_code == 404
+
+        # root member can
+        self.login_as_root_member()
+        resp = self.client.patch(url, params, content_type='application/json')
+        assert resp.status_code == 200
+
+    @skip("doesn't work???")
```
Comment (kflemin) [inline seed/tests/test_property_views_viewset.py:84]: do we still want this one like this?

**Review by kflemin — APPROVED** [review]

File: seed/tests/test_property_views_viewset.py (hunk 1 above)
Comment (haneslinger) [reply seed/tests/test_property_views_viewset.py:84]: I would like it to work. Do you know why it's not?

Comment (haneslinger) [reply seed/tests/test_property_views_viewset.py:84]: @kflemin repinging this. idk why this doesn't work

**Pull request merged**